misc/scripts/mockgen.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
not sure why this file is newly added, did we removed it somehow?

Reply:
it was removed accidentally in a previous PR #1660